From stern@rowland.harvard.edu Wed Feb  1 07:47:21 2006
Date: Wed, 1 Feb 2006 10:47:11 -0500 (EST)
From: Alan Stern <stern@rowland.harvard.edu>
To: Andrew Morton <akpm@osdl.org>
cc: David Brownell <david-b@pacbell.net>, Greg KH <greg@kroah.com>
Subject: usbcore: fix compile error with CONFIG_USB_SUSPEND=n
Message-ID: <Pine.LNX.4.44L0.0602011044270.5635-100000@iolanthe.rowland.org>

This patch (as647) fixes a small error introduced by a recent change to 
the USB core suspend/resume code.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>

---
 drivers/usb/core/hub.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/core/hub.c
+++ gregkh-2.6/drivers/usb/core/hub.c
@@ -1892,8 +1892,8 @@ int usb_resume_device(struct usb_device 
 			status = hub_port_resume(hdev_to_hub(udev->parent),
 					udev->portnum, udev);
 		} else
-			status = 0;
 #endif
+			status = 0;
 	} else
 		status = finish_device_resume(udev);
 	if (status < 0)
